docs/unified-test-documentation/dasharo-compatibility/321-docking-station-usb-c.md

```diff
@@ -51,13 +51,8 @@ Thunderbolt 4 port.
 
 **Expected result**
 
-Output of the command should show that the battery is being charged.
-
-Example output:
-
-```bash
-Charging
-```
+Output of the command should show one of this: `Charging` or `Full`. That means
```

Comment:
I think, that we should leave the example output section. Due to this section, customers might see if the output does not contain any other information, etc.

Reply:
restored example output